src/index.ts

```diff
@@ -45,6 +45,7 @@ const defaultOptions = {
 export default (options: {
   client?: ApolloClient<any>;
   clientOptions?: ApolloClientOptions<any>;
+  debug?: boolean;
```

Comment:
About this new params, do you think its important to keep it for now? I'm open to keep it if we leverage it more and document it in the readme

Reply:
Both works well, they could be added as-it. The debug option is a must have for logging the API response data, otherwise it gets hidden (not possible to log it from my app, must be done from this lib)

If the implementation works for you then let's merge it as-it. Documentation can be updated as well. There is no breaking change.